components/script/dom/bindings/typedarrays.rs

```diff
@@ -31,6 +36,13 @@ where
     T: TypedArrayElement + TypedArrayElementCreator,
     T::Element: Clone + Copy,
 {
+    pub fn new(internal: *mut JSObject) -> HeapTypedArray<T> {
+        HeapTypedArray {
+            internal: Heap::boxed(internal),
+            phantom: PhantomData::default(),
+        }
+    }
```

Comment:
Since this is now unused, let's remove it(we can look at adding something similar in the other PR).

Reply:
I am using it in return of `create_new_external_array_buffer` `HeapTypedArray::<T>::new(array_buffer)`

Reply:
Ok, I think it's better to just create the struct directly in `create_new_external_array_buffer`, because with this you are creating a public API of `HeapTypedArray`, which is unnecessary because you don't need it for now. In your other PR we can look further into adding a kind of `new` method.